modules/ROOT/pages/design-mocking-service.adoc

```diff
@@ -1,68 +1,62 @@
+If you specified a base URI (RAML) or base path (OAS), API Designer enables you to select to use either the base URI or the mocking service when simulating an API call. API Designer does not make any changes to your APIs.
 
-When you activate the mocking service, it exposes a mock service that you can use for testing in lieu of an actual implementation. If you have already specified a base URI (RAML) or base path (OAS), the mocking service comments it out. When you deactivate the mocking service, it removes the base URI or base path that it used and uncomments your base URI or base path. For example, if the base URI in a RAML API specification is `+http://jsonplaceholder.typicode.com+`, the mocking service replaces it temporarily like this, where `+https://anypoint.mulesoft.com/mocking/api/v1/links/+`, etc. is the URI used by the mocking service:
+== Before You Begin
 
-----
-#%RAML 1.0
-title: placeholder
-version: 1.0
-baseUri: https://anypoint.mulesoft.com/mocking/api/v1/links/d0422e05-e779-4016-9d9e-027e50cd15a7
-# baseUri: http://jsonplaceholder.typicode.com
-----
+You must have a project containing an API specification in API Designer. Also, you must define at least one endpoint in your API specification.
 
-You can run the mocking service from the text editor, the visual editor, and from Anypoint Exchange. If you want to run the mocking service from Anypoint Exchange after publishing your API specification, see "To Simulate Data in REST APIs" in the Anypoint Exchange documentation. A link to that topic is in the *See also* section at the end of this topic.
+== Simulate a Call to an API using an Internal URL
 
-[NOTE]
+[IMPORTANT]
 ====
-API Designer's public, mocked URI is also visible in Exchange. There, people who discover your API can try it out by using the built-in Exchange mocking service, or by using the public URI either in their code or in a tool for testing API implementations.
+To access the internal URL, you must specify the authorization heads as part of the request URL. To test an API without supplying authorization headers, enable the external  URL for this project.
 ====
 
-== Procedure
-
-* If you are using the text editor in API Designer:
-+
-. Turn on the mocking service by clicking the *Mocking service* slider in the top-right corner of the editor.
-. In the API summary panel, expand completely the resource with the endpoint that you want to test.
+. From API Designer, open the project containing the API you want to test.
++ 
+You can use either the text editor or the visual editor.
+. Select the root file of your project.
+. Click the *Documentation* icon in the right-hand tool bar.
+. From the drop-down menu, select *Summary*.
+. If you have defined `baseURL` in your API specification:
```

Comment:
Before the step select `baseURL`. Maybe we could add:
* Select a method.

Reply:
Added.